controls/V-219150.rb

```diff
@@ -69,5 +69,15 @@
   tag fix_id: 'F-20874r304779_fix'
   tag cci: ['V-100525', 'SV-109629', 'CCI-001199', 'CCI-002475', 'CCI-002476']
   tag nist: ['SC-28', 'SC-28 (1)', 'SC-28 (1)']
-end
 
+  unless input('is_manual_nondefault_install_partition')
+    describe command('sudo lsblk | grep -A3 `cat /etc/crypttab |cut -d" "  -f1` |grep root | awk "{print $NF}"') do
+      its('exit_status') { should eq 0 }
+      its('stdout.strip') { should eq '/' }
+    end
+  else
+    describe 'Manual test' do
```

Comment:
Two Small issues:
1. We have learned the `describe` text should match the `skip` message so it shows up nicely in heimdall
2. When we `skip` we want to be descriptive so our reporting is as readable as possible and we don't have a lot of unhelpful 'manual review' statements that we then have to dig into and find out what needs to be done.

Can we update this to be something like, `Please review the partition layout to ensure it {{ meets defined conditions }} `